client/src/styles/components/_allcode.scss

```diff
@@ -35,7 +35,7 @@
 	}
 
 	.center {
-		margin: 0 auto;
+		margin: 0 50px;
```

Comment:
Just because I thought it would look better but yeah it doesn't really matter. I can change it back.